app/src/main/java/com/fabulouszanna/pokedex/ui/filters/FilterDialog.kt

```diff
@@ -85,7 +85,7 @@ class FilterDialog(
         filterAdapter = FilterViewPagerAdapter(this, onGenFilterClicked = { currentGen = it } , onTypeFilterClicked = { currentType = it })
         viewPager.apply {
             adapter = filterAdapter
-            this.recyclerView.enforceSingleScrollDirection()
+//            this.recyclerView.enforceSingleScrollDirection()
```

Comment:
commented code is not used code. Git is used for history.

Reply:
Addressed in next commit